package-lock.json

```diff
@@ -17244,6 +17188,103 @@
 				"mkdirp": "bin/cmd.js"
 			}
 		},
+		"node_modules/migrate-mongoose/node_modules/mongodb": {
+			"version": "3.7.4",
+			"resolved": "https://registry.npmjs.org/mongodb/-/mongodb-3.7.4.tgz",
+			"integrity": "sha512-K5q8aBqEXMwWdVNh94UQTwZ6BejVbFhh1uB6c5FKtPE9eUMZPUO3sRZdgIEcHSrAWmxzpG/FeODDKL388sqRmw==",
+			"dependencies": {
+				"bl": "^2.2.1",
+				"bson": "^1.1.4",
+				"denque": "^1.4.1",
+				"optional-require": "^1.1.8",
+				"safe-buffer": "^5.1.2"
+			},
+			"engines": {
+				"node": ">=4"
+			},
+			"optionalDependencies": {
+				"saslprep": "^1.0.0"
+			},
+			"peerDependenciesMeta": {
+				"aws4": {
+					"optional": true
+				},
+				"bson-ext": {
+					"optional": true
+				},
+				"kerberos": {
+					"optional": true
+				},
+				"mongodb-client-encryption": {
+					"optional": true
+				},
+				"mongodb-extjson": {
+					"optional": true
+				},
+				"snappy": {
+					"optional": true
+				}
+			}
+		},
+		"node_modules/migrate-mongoose/node_modules/mongodb/node_modules/optional-require": {
+			"version": "1.1.8",
+			"resolved": "https://registry.npmjs.org/optional-require/-/optional-require-1.1.8.tgz",
+			"integrity": "sha512-jq83qaUb0wNg9Krv1c5OQ+58EK+vHde6aBPzLvPPqJm89UQWsvSuFy9X/OSNJnFeSOKo7btE0n8Nl2+nE+z5nA==",
+			"dependencies": {
+				"require-at": "^1.0.6"
+			},
+			"engines": {
+				"node": ">=4"
+			}
+		},
+		"node_modules/migrate-mongoose/node_modules/mongoose": {
+			"version": "5.13.21",
```

Comment:
we have to check this.... migrate-mongoose uses its own older version of mongoose...
they both eventually use the same DB server, but require diferent versions

Reply:
we will replace this package in other Ticket